pkg/vulnsrc/amazon/amazon_test.go

```diff
@@ -330,57 +254,34 @@ func TestConfig_WalkFunc(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
-		zc, recorder := observer.New(zapcore.DebugLevel)
```

Comment:
Curious as to why was asserting logging was removed? If we logging is important (for debugging, testing or other purposes) it should be considered part of core business logic and should be tested appropriately.

Reply:
I agree with you. But your commit regarding `Logger` causes a nil panic since Logger needs to be initialized before use. To sort it out, we need more fixes. If we do it, the implementation of Amazon is quite different from other OSes. I think it is confusing. 
So, I decided to open a different PR for Logger. At that time, I consider it is better to change the implementation of all OSes.